.github/workflows/ci.yml

```diff
@@ -19,18 +19,44 @@ env:
 
 jobs:
   c9s-bootc-e2e:
-    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        runner:
+          - ubuntu-24.04
+          - ubuntu-24.04-arm
+
+    runs-on: [ "${{ matrix.runner }}" ]
+
     steps:
       - uses: actions/checkout@v4
+
+      - name: Install podman
+        if: ( matrix.runner == 'ubuntu-24.04-arm' )
+        run: |
+          sudo apt update -y
+          sudo apt install -y podman
+          sudo podman build -t localhost/bootupd:latest -f ci/Containerfile.c9s .
```

Comment:
Cool, overall LGTM, just minor comment, this line is duplicated in `build` step, maybe should remove this line.